.mocharc.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+
+const nodeOptions = ["experimental-vm-modules"];
+const [major, _minor, _patch] = process.versions.node.split('.');
```

Comment:
```suggestion
const [major] = process.versions.node.split('.');
```

No need to keep those around